plugins/attachment/src/index.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright © 2021 Anticrm Platform Contributors.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Class, Doc, Ref } from '@anticrm/core'
+import type { Plugin, Service } from '@anticrm/platform'
+import { plugin } from '@anticrm/platform'
+import type { IntlString } from '@anticrm/status'
+
+export interface Attachment extends Doc {
+  objectId: Ref<Doc>
+  objectClass: Ref<Class<Doc>>
+  name: string
+  format: string
+  size: number
+}
+
+export function nameToFormat (name: string): string {
+  return name.slice(name.lastIndexOf('.') + 1)
+}
+
+export function sizeToString (size: number): string {
+  const sizes = ['bytes', 'KB', 'MB', 'GB']
+  let i = 0
+
+  for (i = 0; i < sizes.length; i++) {
+    if (size < 1024) break
+    size = size / 1024
```

Comment:
What you mean?

Reply:
Personally, I don't like mess of units that still exists, SI prefixes have decimal base, in case we use binary base there are different prefixes for that.